client/src/contexts/ListContext.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+const ListContext = React.createContext({});
+
+export function ListStore(props) {
+const [listChange, setListChange] = useState(true);
+
+const listToggle = () => {
+    setListChange(!listChange);
+}
```

Comment:
What is the purpose of this? I don't see it being used elsewhere except one call of this function.
If we want to use context, it should be for all of the list data and API calls, but I think we can do without the context at all since every component that needs access to list data should have a shared parent component of the dashboard